packages/go_router/lib/src/router.dart

```diff
@@ -83,6 +83,7 @@ class GoRouter implements RouterConfig<RouteMatchList> {
     bool debugLogDiagnostics = false,
     GlobalKey<NavigatorState>? navigatorKey,
     String? restorationScopeId,
+    bool? requestFocus,
```

Comment:
can you add a doc comment in gorouter describe this parameter

Reply:
why is this nullable? I think we can default it to true?

Reply:
done